cmd/git-init/main.go

```diff
@@ -34,6 +33,7 @@ func init() {
 	flag.StringVar(&fetchSpec.URL, "url", "", "Git origin URL to fetch")
 	flag.StringVar(&fetchSpec.Revision, "revision", "", "The Git revision to make the repository HEAD")
 	flag.StringVar(&fetchSpec.Path, "path", "", "Path of directory under which Git repository will be copied")
+	flag.BoolVar(&fetchSpec.SSLVerify, "sslVerify", false, "Enable/Disable SSL verification in the git config")
```

Comment:
should the default here be `true`?

Reply:
Ahh ... yes .... let me make a modification, I'll need to also pass through =true/=false from git_resource.go for this to default to true here ..... but that should be no bother.  I'll take a look at failing unittest too.